packages/cli/src/cmds/new.ts

```diff
@@ -56,7 +47,7 @@ export default class New extends YargsCommand {
         .options({
           interactive: {
             description: "Select the options interactively",
-            boolean: true,
+            boolean: booleanParamNames.has("interactive"),
             default: true,
```

Comment:
I think `true` is enough. Because if you let `boolean` be from `booleanParamNames`, you also need to let `default` be as it.